.github/workflows/add-discourse-topic.yml

```diff
@@ -0,0 +1,30 @@
+name: Add discourse topic
+on:
+  workflow_dispatch:
+jobs:
+  add-discourse-topic:
+    runs-on: ubuntu-latest
+    steps:
+
+      # Get the branch
+      - name: Get branch
+        uses: actions/checkout@v3
+
+      # Identify the action
+      - name: Identify the action
+        run: |
+          git config user.name "GitHub Action (Add discourse topic)"
+          git config user.email "<support@dockstore.org>"
+
+      # Add any missing discourse topics
+      - name: Add topics
+        env:
+          DISCOURSE_API_KEY: ${{ secrets.DISCOURSE_API_KEY }}
+        run: |
+          ./add-discourse-topic-to-all.sh
+
+      # Commit and push, should do nothing if there were no changes.
+      - name: Commit and push
+        run: |
+          git commit -a -m "add discourse topic"
```

Comment:
Is it possible to add checks so that commits aren't made directly to develop nor master?

I don't know if that's necessary, perhaps branch protection rules will kick in -- I'm don't know what permissions this action has.

Reply:
I added a step that checks the branch and aborts if it's `develop` or `master`.